examples/python-cicd-with-cloudbuilder/cloudbuild.yaml

```diff
@@ -0,0 +1,13 @@
+steps:
+- name: 'gcr.io/$PROJECT_ID/python-cloudbuild'
```

Comment:
This is more of a question -- does the user need to set the $PROJECT_ID or does google deduce this from the gcloud builds command? I assume it is automatically populated when run from a build trigger. May be worth noting this when you document the steps in the yaml.

Reply:
GCP Cloud Build deduces it.  I'll note that in the doc